src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs

```diff
@@ -50,7 +50,7 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
                     if (jsonResponse.Resource.rate_limit is 1)
                     {
-                        _logger.Debug("No results due to rate limiting. Reason: {0}", reason);
+                        _logger.Info("No results due to rate limiting. Reason: {0}", reason);
```

Comment:
Actually should this be a warning instead?

Reply:
As much as they rate limit, no, I don't think so

Reply:
should we be throwing ratelimitreached or is https://github.com/Prowlarr/Prowlarr/issues/1277 still occurring?